src/network/mcpe/handler/DeathPacketHandler.php

```diff
@@ -57,6 +58,11 @@ public function handlePlayerAction(PlayerActionPacket $packet) : bool{
 		return false;
 	}
 
+	public function handleContainerClose(ContainerClosePacket $packet): bool
```

Comment:
In accord with PocketMine's code formatting, here you should put a space between ")" and ":" and put the "{" on the same line as the function declaration.

Reply:
Fixed! thanks for letting me know